kpops/utils/yaml.py

```diff
@@ -121,6 +121,7 @@ def print_yaml(data: Mapping | str, *, substitution: dict | None = None) -> None
     """
     if not isinstance(data, str):
         data = yaml.safe_dump(dict(data))
+        data = f"---\n{data}"
```

Comment:
technically there's also `yaml.safe_dump_all` for a sequence of YAML documentes but I am fine with merging it like this